internal/dflog/logcore.go

```diff
@@ -52,12 +58,28 @@ var customCoreLevel atomic.Bool
 var grpcLevel = zap.NewAtomicLevelAt(zapcore.WarnLevel)
 var customGrpcLevel atomic.Bool
 
-func CreateLogger(filePath string, compress bool, stats bool, verbose bool) (*zap.Logger, zap.AtomicLevel, error) {
+func CreateLogger(filePath string, compress bool, stats bool, verbose bool, config LogRotateConfig) (*zap.Logger, zap.AtomicLevel, error) {
+
+	maxSize := defaultRotateMaxSize
```

Comment:
Set default value in each service's config. 
Scheduler: https://github.com/dragonflyoss/Dragonfly2/blob/main/scheduler/config/constants.go.
Manager: https://github.com/dragonflyoss/Dragonfly2/blob/main/manager/config/constants.go.
Client: https://github.com/dragonflyoss/Dragonfly2/blob/main/client/config/constants.go.

Reply:
Sure, this is done now. One question, I noticed `logger.InitDaemon()` is called from `SetupDaemon()` in `log.go`. But I cannot see `SetupDaemon()` being called anywhere within `Dragonfly2`. What is the purpose of `SetupDaemon()`?

Reply:
If there are other services that want to use dragonfly's log, they will be initialized through this `SetupDaemon`.

Reply:
The `SetupDaemon` is deprecated, we will clean up it, resolved here.